webapp/views.py

```diff
@@ -69,20 +74,55 @@ def _normalise_resources(self, resources):
 
     def _get_resources(self):
         topic = self.request.GET.get('topic')
+        content = self.request.GET.get('content')
         feed_items = {}
-        if topic:
-            category_id = str(self.GROUPS[topic]['id'])
-            api_url = (
-                '{api_url}/posts?_embed&group={category_id}&{resource_filter}'
-            )
-            api_url = api_url.format(
-                api_url=self.API_URL,
-                category_id=category_id,
-                resource_filter=self.RESOURCE_FILTER,
-            )
-            feed_items[topic] = {}
-            feed_items[topic]['items'] = get_json_feed_content(api_url)
-            feed_items[topic]['group_name'] = self.GROUPS[topic]['name']
+        if topic or content:
+            if not content:
+                category_id = str(self.GROUPS[topic]['id'])
+                api_url = (
+                    '{api_url}/posts?_embed'
+                    '&group={category_id}&{resource_filter}'
```

Comment:
Do the ampersands in the urls need to be escaped?

Reply:
No, they are a string that represent the URI to the data endpoint.